content/rs/databases/active-active/_index.md

```diff
@@ -27,121 +27,44 @@ Active-Active databases do not replicate the entire database, only the data.
 Database configurations, LUA scripts, and other support info are not replicated.
 {{< /note >}}
 
-You can create Active-Active databases on Redis Enterprise Software or Redis Cloud.
+## High availability
 
-## Considerations for Active-Active databases {#considerations-for-activeactive-databases}
+The [high availability]({{<relref "/rs/databases/configure/durability-ha.md">}}) that Active-Active replication provides is built upon a number of Redis Enterprise Software’s features (such as [clustering]({{<relref "/rs/databases/configure/clustering.md">}}), [replication]({{<relref "/rs/databases/configure/replication.md">}}), and [replicaHA]({{<relref "/rs/databases/configure/replica-ha.md">}})) as well as some features unique to Active-Active ([multi-primary replication]({{<relref "#multi-primary-replication/">}}), [automatic conflict resolution]({{<relref "#conflict-resolution/">}}), and [strong eventual consistency]({{<relref "#strong-eventual-consistency/">}})).
 
-Active-Active databases are based on multi-master replication that is configured to run on each database.
-An Active-Active database is made up of instances of the data that are each stored on a Redis Enterprise cluster.
+Clustering and replication are used together in Active-Active databases to distribute multiple copies of the dataset across multiple nodes and multiple clusters. This helps reduce the risk of a node or cluster becoming a single point of failure. If a primary node or primary shard fails, a replica will automatically be promoted to primary. The [replicaHA]({{<relref "/rs/databases/configure/replica-ha.md">}}) feature (enabled by default), will automatically migrate replica shards to available nodes to avoid one node holding all copies of certain data.
```

Comment:
```suggestion
Clustering and replication are used together in Active-Active databases to distribute multiple copies of the dataset across multiple nodes and multiple clusters. This helps reduce the risk of a node or cluster becoming a single point of failure. If a primary node or primary shard fails, a replica will automatically be promoted to primary. The [replica HA]({{<relref "/rs/databases/configure/replica-ha.md">}}) feature (enabled by default), will automatically migrate replica shards to available nodes to avoid one node holding all copies of certain data.
```
Not sure if this is a typo or not.